pandas/core/frame.py

```diff
@@ -3835,34 +3835,87 @@ def sortlevel(self, level=0, axis=0, ascending=True, inplace=False,
                                inplace=inplace, sort_remaining=sort_remaining)
 
     def nlargest(self, n, columns, keep='first'):
-        """Get the rows of a DataFrame sorted by the `n` largest
-        values of `columns`.
+        """
+        Return the `n` first rows ordered by `columns` in descending order.
+
+        Return the `n` first rows with the largest values in `columns`, in
+        descending order. The columns that are not specified are returned as
+        well, but not used for ordering.
```

Comment:
If I'm right, this is equivalent to using `df.sort_values(columns, ascending=False).head(n)`, isn't it? If that's the case, I'd explicitly say it in the extended summary, and I'd add both methods to the `See Also` section.

Reply:
it’s an equivalent result, but this is much more performany